osquery/tables/system/windows/cpu_info.cpp

```diff
@@ -57,6 +57,9 @@ QueryData genCpuInfo(QueryContext& context) {
     (data.GetLong("MaxClockSpeed", number))
         ? r["max_clock_speed"] = INTEGER(number)
         : r["max_clock_speed"] = "-1";
+    (data.GetLong("LoadPercentage", number))
```

Comment:
This ternary ends up returning `-1` when the actual load percentage reported is `0`.

Reply:
No, actually that doesn't seem to be the case as I've now seen it successfully return `0`. Not clear exactly why it returns `-1` sometimes.

Reply:
Okay I manually ran the WMI query and verified that it sometimes just doesn't return a value.

```
Get-WmiObject -Query 'Select LoadPercentage from Win32_Processor'
```

Reply:
I suspect sometimes WMI is locked or weird. It all feels like heavy RPC stuff